tensorflow/python/ops/image_ops_impl.py

```diff
@@ -790,8 +790,17 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
 
   Raises:
     ValueError: If the shape of `image` is incompatible with the `offset_*` or
-      `target_*` arguments, or either `offset_height` or `offset_width` is
-      negative.
+    `target_*` arguments, or either `offset_height` or `offset_width` is
+    negative.
+      
+  Usage example:
+  
+  python
+    
+    >>> x = tf.random.uniform((28, 28, 1))
+    >>> tf.image.pad_to_bounding_box(x, 28, 28, 100, 100).shape
+    >>> TensorShape([100, 100, 1])
```

Comment:
Why are these here?

Reply:
Its the usage example

Reply:
But I think you might be referring to the `TensorShape([100, 100, 1])`... Umm in this commit I removed the `>>>` that comes before it, so everyone knows its the output.